src/pages/Components/Tags/design.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Design'
+order: 4
+---
+
+## Tags sketch file is composed of three sections :
+
+The sketch file includes other form elements. Here is the section you'll find the radio.
+
+### **👾 For developers :**
+
+This part is a specific section for developers, **for a better accessibility**. There are all tag states : default, hover, focus, active, disable, etc.
+
+### **🛠 ToolKit :**
+
+The ToolKit section bring together the **dedicated symbols used for building the final symbols**, and you can find them in the symbol overrides.
```

Comment:
The ToolKit section contain the **symbols used to compose the final symbols**,  and they are not meant to be used directly but are meant to be accessible through symbol overrides.